### DISCUSSION TITLE
iPXE initializing stuck - Lenovo Yoga 370

### DISCUSSION THREAD

**Discussion by leofuehringer** [ipxe/ipxe#1359] [General]
Hello,
The pxe boot works on most laptops (L13 Yoga, Fujitsu Lifebook A3510, etc.) Unfortunately it hangs on some Lenovo Yoga 370s. It works on the 20JH002FUS, BIOS Version 1.13, Serial-Number: MP18ZKFP
Unfortunately not with the 20JJS01S1D, BIOS Version 1.34, Serial-Number: MP186QDG

![pxe-boot](https://github.com/user-attachments/assets/6936f65b-97f2-46cc-979e-337e655b19e9)

I can't find anything unusual in the server's syslog.

Does anyone have an idea?

Thanks

**Comment by NiKiZe** [comment]
Just created #1358
Also see https://github.com/ipxe/ipxe/discussions/1348#discussioncomment-11507064

**Reply by NiKiZe** [reply]
The picture shows you get further than what was found in linked issues.

**Comment by leofuehringer** [comment]
I don't think my problem fits into the problems mentioned. I added a picture in the first post. Unfortunately nothing happens from here. Ctrl+Alt+Del doesn't work either

**Reply by NiKiZe** [reply]
Have you tried with both `ipxe.efi` and `snponly.efi` ?

Do you have anything in your autoexec.ipxe that would provide any early output?

**Reply by NiKiZe** [reply]
Are you and the device still available for testing? or did it work with snponly?

**Comment by ShaowulfD** [comment]
We are having the same issue with Surface Pro 7 and Surface Pro 9 as well as the Lenovo X1 Carbon Gen 10 and Gen 11 

happened after a firmware update

What are you PXE booting into?  Curious if we are using the same appliance, we are trying to boot into our KACE Systems Deployment Appliance

![Surface9_PXE Error](https://github.com/user-attachments/assets/a780add1-5465-48c5-af99-92b8e11ddd9a)

**Reply by NiKiZe** [reply]
I know this is old, but this picture shows hang before the iPXE version banner, which is consistent with some other issues (see my hidden post above for some of them)
